node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,8 +135,10 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            var supportedTypes = _.union(_.pluck(navigator.mimeTypes, 'type'), ['text/html', 'text/plain']);
```

Comment:
Regardless, this would need a line of JSDocs and an `@see` for `navigator.mimeTypes`